policy_packs/aws/backup/enforce_ebs_volume_backups/README.md

```diff
@@ -0,0 +1,160 @@
+---
+categories: ["data protection"]
+primary_category: "data protection"
+---
+
+# Enforce Backups of EBS Volumes
+
+Enforcing backups of EBS volumes provides a crucial defense against lost data. Guardrails uses the AWS Backup service to create and manage backups of EBS volumes. 
+
+This [policy pack](https://turbot.com/guardrails/docs/concepts/policy-packs) can help you configure the following settings to create the required Backup IAM role, Backup Vaults, Backup Plans and Backup Selections. 
```

Comment:
Can you please use a list format like we do in other READMEs, e.g., https://github.com/turbot/guardrails-samples/tree/main/policy_packs/aws/ec2/enforce_approved_amis_publishers_for_instances, https://github.com/turbot/guardrails-samples/tree/main/policy_packs/aws/kms/enforce_rotation_is_enabled_for_keys.

You can have the sentence be more generic like `This policy pack can help you configure:` since it's creating so many different types of resources